reports/update-orphaning/Update orphaning analysis using longitudinal dataset.ipynb

```diff
@@ -79,7 +80,12 @@
    },
    "outputs": [],
    "source": [
-    "today = dt.date.today()\n",
+    "today_env_str = environ.get(\"date\", None)\n",
+    "assert (today_env_str is not None), \"The date environment parameter is missing.\"\n",
+    "\n",
+    "# Uncomment the next line to run manually\n",
+    "#today = dt.date.today()\n",
+    "today = dt.datetime.strptime(today_env_str, \"%Y%m%d\")\n",
     "\n",
     "# Find the date of last Wednesday to get the proper 7 day range, and to find the last longitudinal dataset correctly.\n",
```

Comment:
Note that you can't rely any longer on `date` being a Wednesday and the variable names and the code should reflect that. Airflow might run this job at any time.